src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2628,6 +2663,7 @@ private string GetCurrentDirectoryForConditionEvaluation(ProjectElement element)
         {
             if (element is ProjectPropertyGroupElement || element is ProjectImportElement || element is ProjectImportGroupElement)
             {
+                //  TODO: If it's a ProjectImportElement with an Sdk attribute, then resolve the SDK and return its path (don't throw an error if it doesn't exist, though)
```

Comment:
The idea is that the method should return `ResolveSdk( ((ProjectImportElement)element).Sdk, ((ProjectImportElement)element).SdkLocation )`.  This will let you write import statements like this:

```xml
<Import Project="Sdk.props" Sdk="MySdk/1.0.0" Condition="Exists('Sdk.props')" />
```

I was also thinking `ResolveSdk` should have an overload to not throw if the Sdk isn't found, so that the condition can be evaluated and return false in this case.